packages/web-frontend/src/components/View/SingleValueWrapper.js

```diff
@@ -1,57 +1,46 @@
-/**
- * Tupaia Web
- * Copyright (c) 2019 Beyond Essential Systems Pty Ltd.
- * This source code is licensed under the AGPL-3.0 license
- * found in the LICENSE file in the root directory of this source tree.
+/*
+ * Tupaia
+ * Copyright (c) 2017 - 2022 Beyond Essential Systems Pty Ltd
  */
-/**
- * SingleValueWrapper
- *
- * Renders view with single value from data provided by viewContent object
- * @prop {object} viewContent An object with the following structure
```

Comment:
If we use prop types correctly then we don't also need this doc block and just having the one place with the shape of the viewContent is easier to maintain.